foundation/foundation-configuration-store/src/test/resources/specifications/load-configurations.feature

```diff
@@ -69,3 +69,16 @@ Feature: Load configurations at specified URI based on environment context
     Given there exists a krausening_password and encrypted properties
     When the configuration service starts
     Then the loaded properties contains the decrypted value
+
+  Scenario: The ConfigLoader can read and write to a specific dao type
+    Given the config loader to configured to use "inMemory"
+    And the properties are loaded
+    When the configurations are written
+    Then the properties can be read
+
+  @vault @integration
+  Scenario: The ConfigLoader can read and write to the Vault server
+    Given the config loader to configured to use "vault"
+    And the properties are loaded
+    When the configurations are written
+    Then the properties can be read
```

Comment:
Q: Do we need one for `Given the config loader to configured to use "krausening"` or is this covered in the other scenarios?

Reply:
Yeah I should add that. Looking at it, Ill refactor the tests to pull the test resources from target so any changes during the updates are not saved into the code base